dsl/properties/scripts/DockerClient.groovy

```diff
@@ -860,8 +860,13 @@ public class DockerClient extends BaseClient {
     def attachAdditionalNetworks(def container, def serviceDetails){
         def networkList = getNetworkList(serviceDetails)
 
-        for(int i=1;i<networkList.size();i++){
-            dockerClient.connectNetwork(networkList[i], container)
+        for(int i=0;i<networkList.size();i++){
+            try{
+                    dockerClient.connectNetwork(networkList[i], container)
```

Comment:
Is there a way to distinguish between a regular exception and the one thrown if the network is already attached? We do not want to eat up any other exception.